.travis.yml

```diff
@@ -2,8 +2,8 @@ sudo: required
 
 language: ruby
 rvm:
-  - 2.2
-  - 2.3.0
+  - 2.3.3
```

Comment:
Are we ready to drop support for 2.2.?
I think ManageIQ supports 2.2.2 at minimum
http://manageiq.org/docs/guides/developer_setup#install-ruby-and-bundler